server/app/services/migration/ProgramMigrationService.java

```diff
@@ -129,4 +130,33 @@ public String maybeGenerateNewAdminName(String adminName) {
     }
     return newAdminName;
   }
+
+  /**
+   * Prepare the {@link ProgramDefinition} for export by making any changes required, such as
+   * removing settings that may be specific to an environment.
+   *
+   * @param programDefinition the {@link ProgramDefinition} to modify
+   * @return a modified {@link ProgramDefinition} ready for export
+   */
+  public ProgramDefinition prepForExport(ProgramDefinition programDefinition) {
+    return programDefinition.toBuilder()
+        // TODO(#7087) migrate program categories
+        .setCategories(ImmutableList.of())
+        // Don't export environment specific notification preferences
+        .setNotificationPreferences(ImmutableList.of())
```

Comment:
In staging you might have emails set differently for testing than you'd want them in prod.

In particular, if an admin turned them off in staging, I want admins to have to deliberately turn them off in prod too so that they don't miss any applications. That said, I don't feel strongly and can change this in a followup.